cpp/tests/reductions/scan_tests.cpp

```diff
@@ -135,8 +135,18 @@ struct ScanTest : public BaseScanTest<T> {
       switch (agg->kind) {
         case aggregation::SUM: return HostType{0};
         case aggregation::PRODUCT: return HostType{1};
-        case aggregation::MIN: return std::numeric_limits<HostType>::max();
-        case aggregation::MAX: return std::numeric_limits<HostType>::lowest();
+        case aggregation::MIN:
+          if constexpr (cuda::std::numeric_limits<HostType>::has_infinity) {
+            return cuda::std::numeric_limits<HostType>::infinity();
+          } else {
+            return std::numeric_limits<HostType>::max();
+          }
```

Comment:
Following up on the previous comment -- should we be using `cuda::std` or just `std`? We should be consistent here.

Reply:
```suggestion
          if constexpr (std::numeric_limits<HostType>::has_infinity) {
            return std::numeric_limits<HostType>::infinity();
          } else {
            return std::numeric_limits<HostType>::max();
          }
```